examples/sampling/graphbolt/node_classification.py

```diff
@@ -348,6 +382,8 @@ def main(args):
     out_size = num_classes
 
     model = SAGE(in_size, hidden_size, out_size)
+    assert len(args.fanout) == len(model.layers)
```

Comment:
I have a question that why we need to explicitly assert equal? Does we have any other way to check it silently / automatically?

Reply:
As discussed before, it is hard to check automatically since the structure of model is defined by users.

Reply:
Got